optimizer_wrapper.rb

```diff
@@ -808,9 +814,7 @@ def self.apply_zones(vrp)
 
         next unless zone.inside(activity_loc.lat, activity_loc.lon)
 
-        service.sticky_vehicles += zone.vehicles
-        service.sticky_vehicles.uniq!
-        service.skills += [zone[:id]]
+        service.skills += [zone[:id].to_sym]
```

Comment:
can't we make the zone ID's symbol at the API level so that we don't have this to_sym business?